pandas/core/arrays/string_.py

```diff
@@ -104,6 +104,11 @@ class StringArray(PandasArray):
 
     .. versionadded:: 1.0.0
 
+    .. versionchanged:: 1.1.0
+
+        ``StringArray`` allow non-string input values, but will always convert the
+        values  to strings. (Before Pandas 1.1 non-string values were not allowed).
```

Comment:
This is not really a change in the `StringArray` constructor (that still disallows non-strings), so I would leave out the versionchanged note here

Reply:
Yeah, deleted.